mobify/mobifyjs
# Make Tag Backward Compatible

**Status**: _Opened for Visibility_

**Reviewers**: @jansepar 
**Linked PRs**: https://github.com/mobify/portal_app/pull/2203 (This has to be deployed before we roll out tags)
## Changes
- Adds tests for v1.0 Projects
  - Fixes issue on v1.0 projects (Mobify.tagVersion must exist)
- Adds test for v1.1 Projects
- Adds test for v2.0 Projects
- Adds test that loads tag in old browser, and ensures it does not blow up.
  - Fixes issue with `console.log` and `Date.new` not existing on IE 
- Adds support for preview
## How to test-drive this PR
- Run the tests, and then run them on sauce labs.
- Use Charles to inject the tag in a page (eg, from a CST project) and map preview.mobify.com to your local portal_app
  - Ensure that old projects load (1.0, 1.1, MPS)
  - Ensure you can preview
